workerapi/worker.thrift

```diff
@@ -0,0 +1,38 @@
+enum Status {
+  UNKNOWN = 0
+  READY = 1
+  PENDING = 2
+  FAILURE = 3
+  RUNNING = 4
+  SUCCESS = 5
+  ABORTED = 6
+  TIMEOUT = 7
+}
+
+struct WorkerStatus {
```

Comment:
I see them as the same thing, the worker is the task. If we decide we want meta info about the worker that's independent of task, then we need to decide what that info is. I can't think of anything useful for MVP.